packages/waku/src/lib/plugins/vite-plugin-rsc-rsdw.ts

```diff
@@ -41,20 +41,21 @@ export function rscRsdwPlugin(): Plugin {
       mode = env.mode;
     },
     async resolveId(id, importer, options) {
-      if (id === 'react-server-dom-webpack/client.edge') {
+      if (id === 'react-server-dom-vite/client.edge') {
         const resolved = await this.resolve(id, importer, options);
         if (resolved) {
           id = resolved.id;
         }
       }
-      if (id.endsWith('/react-server-dom-webpack/client.edge.js')) {
+      if (id.endsWith('/react-server-dom-vite/client.edge.js')) {
         id =
           id.slice(0, -'/client.edge.js'.length) +
-          `/cjs/react-server-dom-webpack-client.edge.${mode === 'production' ? 'production' : 'development'}.js`;
+          `/cjs/react-server-dom-vite-client.edge.${mode === 'production' ? 'production' : 'development'}.js`;
         return this.resolve(id, importer, options);
       }
     },
     transform(code, id) {
+      if (1) return;
```

Comment:
I thought we could remove this plugin completely. What's blocking?

Reply:
Yes, this is no-op and should be removed. This was just me testing out quickly and auto find&replace modified this file.

Reply:
Alright!
We should remove it when this PR is ready and before merging, or a follow-up PR would do.